### PR TITLE
Simplify home page and admin page patterns

### PR DESCRIPTION
**Pull request by jeffsee55**
This PR moves a lot of things around, trying to simplify the initial concepts and then provide a path for a deeper dive in other areas (the posts page). I think it still needs some work, but I've tried to make the code more obvious about where things can be deleted.

Another thing done in this setup is the `_app.js` loads Tina dynamically. I could go either way on this, but if people don't like it we can go back to what we were doing before, it's just a lot of boilerplate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +29,10 @@ Basic TinaCMS starter based on [Next.js](https://nextjs.org) and [TinaCMS](https
 - 📐 [Content Modeling](#content-modeling)
 - 💡 [Local development workflow tips](#local-development-workflow-tips)
 
+## What is this?
+
+This is a [TinaCMS](https://tinacms.org)-enabled Next.js app, so you can edit your content on a live page. If you've seen a TinaCMS project before, this one might look a little bit different. On this project we're using the Tina file-based CMS via GraphQL. It's powered by a schema that you define, and with that it not only serves content from markdown files in your repo, but it also generates TinaCMS forms for you automatically. You can read more about why we built this [here](), otherwise, let's get started...
```
Comment (DirtyF) [inline README.md:34]:
```suggestion
This is a [TinaCMS](https://tina.io)-enabled Next.js app, so you can edit your content on a live page. In this project the Tina file-based CMS is used via GraphQL: it's powered by a schema that _you_ define.  It not only serves content from Markdown files in your repository, but it also generates TinaCMS forms for you automatically ✨.

 Feel free to read more about [why we built this](). Now, let's get started!
```

File: .tina/schema.ts [hunk 2]
```diff
@@ -1,50 +1,102 @@
 import { defineSchema } from "tina-graphql-gateway-cli";
 
 export default defineSchema({
-  sections: [
+  collections: [
```
Comment (Enigmatical) [inline .tina/schema.ts:4]: 🎉

File: README.md [hunk 3]
```diff
@@ -251,9 +258,9 @@ export default defineSchema({
 
 Be sure this is your default export from this file, we'll validate the schema and build out the GraphQL API with it.
 
-### `sections`
+### `collections`
 
-The top-level key in the schema is an array of sections, a `section` informs the API about _where_ to save content. You can see from the example that a `pages` document would be stored in `content/pages`, and it can be the shape of any `template` from the `templates` key.
+The top-level key in the schema is an array of _collections_, a `collection` informs the API about _where_ to save content. You can see from the example that a `pages` document would be stored in `content/pages`, and it can be the shape of any `template` from the `templates` key.
```
Comment (Enigmatical) [inline README.md:263]: I wonder if the example schema here should avoid referencing a `pages` collection in the same way the starter's `schema.ts` was changed.

**Review by Enigmatical — COMMENTED** [review]
Overall, I really like the little informational and casual, conversational comments throughout the code.  Definitely an improvement over the initial starter. 👍

**Comment by mitchmac** [comment]
@jeffsee55 Can this be merged or waiting on anything?

**Pull request merged**